src/sentry/assistant/guides.py

```diff
@@ -28,6 +28,7 @@
     "release_adoption": {"id": 13, "required_targets": ["release_adoption"]},
     "user_misery": {"id": 14, "required_targets": ["user_misery"]},
     "stack_trace_preview": {"id": 15, "required_targets": ["issue_stream_title"]},
+    "trace_view": {"id": 16, "required_targets": []},
```

Comment:
Why are required targets required below as `['trace_view_guide_row', 'trace_view_guide_row_details']` but none here?

Reply:
Good catch, fixed in be26d44